features/collection/snmp-collector/src/main/java/org/opennms/netmgt/collectd/EnumLookupPropertyExtender.java

```diff
@@ -0,0 +1,91 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2020 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2016 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
```

Comment:
```suggestion
 * along with OpenNMS(R). If you did not, see:
```

Reply:
This one and the others I haven't committed are boilerplate comment headers. If we're going to mount an effort to improve these blocks, I think it ought to be across the whole corpus of code. I'm nervous about changing it just here and there, since some code-analysis tools might deduct points for inconsistent application of license and copyright header.

Reply:
[Here](https://www.gnu.org/licenses/agpl-3.0.html#howto) is the reference copy of the Affero GPLv3 copyright boilerplate block. I think it's best if we stick with the one put out by the FSF.

Reply:
@jeffgdotorg no worries. Context is everything. No need to make the changes.